pom.xml

```diff
@@ -95,7 +95,7 @@
         <dependency>
             <groupId>fr.inria.gforge.spoon</groupId>
             <artifactId>spoon-core</artifactId>
-            <version>9.1.0-beta-11</version>
+            <version>9.1.0</version>
```

Comment:
@slarse Any idea why @renovate-bot did not suggest Spoon version 10.0.0? It is the latest stable release of spoon.

Reply:
Generally, major releases induce breaking behaviour in projects. Probably that's why renovate didn't submit a PR for it.